packages/react-components/react-progress/src/index.ts

```diff
@@ -1,2 +1,3 @@
 // TODO: replace with real exports
 export {};
+export * from './Progress';
```

Comment:
The exports from `src/index.ts` need to be explicitly named (no * exports).

See [Spinner](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-spinner/src/index.ts) for an example of how this needs to look.

Reply:
Got it! I updated the exports!